test/system/jsdoc-config.test.js

```diff
@@ -37,63 +36,63 @@ describe('JSDoc configuration', function () {
 
     describe('source', function () {
         it('should have an include pattern', function () {
-            expect(json.source.includePattern).to.be('.+\\.js(doc)?$');
+            expect(json.source.includePattern).to.equal('.+\\.js(doc)?$');
         });
 
         it('should have an exclude pattern', function () {
-            expect(json.source.excludePattern).to.be('(^|\\/|\\\\)_');
+            expect(json.source.excludePattern).to.equal('(^|\\/|\\\\)_');
         });
     });
 
     describe('plugins', function () {
         it('should have the markdown plugin', function () {
-            expect(_.includes(json.plugins, 'plugins/markdown')).to.be.ok();
+            expect(_.includes(json.plugins, 'plugins/markdown')).to.be.ok;
         });
     });
 
     describe('templates', function () {
         it('should not have clever links', function () {
-            expect(json.templates.cleverLinks).to.not.be.ok();
+            expect(json.templates.cleverLinks).to.not.be.ok;
         });
 
         it('should not have monospace links', function () {
-            expect(json.templates.monospaceLinks).to.not.be.ok();
+            expect(json.templates.monospaceLinks).to.not.be.ok;
         });
 
         it('should highlight tutorial code', function () {
-            expect(json.templates.highlightTutorialCode).to.be.ok();
+            expect(json.templates.highlightTutorialCode).to.be.ok;
         });
     });
 
     describe('opts', function () {
         it('should use the Postman JSDoc theme', function () {
-            expect(json.opts.template).to.be('./node_modules/postman-jsdoc-theme');
+            expect(json.opts.template).to.equal('./node_modules/postman-jsdoc-theme');
         });
 
         it('should use UTF-8 encoding', function () {
-            expect(json.opts.encoding).to.be('utf8');
+            expect(json.opts.encoding).to.equal('utf8');
         });
 
         it('should create documentation in out/docs', function () {
-            expect(json.opts.destination).to.be('./out/docs');
+            expect(json.opts.destination).to.equal('./out/docs');
         });
 
         it('should recurse', function () {
-            expect(json.opts.recurse).to.be.ok();
+            expect(json.opts.recurse).to.be.ok;
         });
 
         it('should have a valid readme', function () {
-            expect(json.opts.readme).to.be('README.md');
+            expect(json.opts.readme).to.equal('README.md');
         });
     });
 
     describe('markdown', function () {
         it('should have a gfm parser', function () {
-            expect(json.markdown.parser).to.be('gfm');
+            expect(json.markdown.parser).to.equal('gfm');
```

Comment:
`expect(json.markdown.parser, 'Should use the gfm markdown parser').to.equal('gfm');`